includes/attendee/attendee-repository.php

```diff
@@ -26,6 +26,27 @@ public function insert_attendee( Attendee $attendee ): void {
 		// phpcs:enable
 	}
 
+	/**
+	 * Update an attendee.
+	 *
+	 * @param Attendee $attendee The attendee to update.
+	 * @return void
+	 */
+	public function update_attendee( Attendee $attendee ): void {
+		global $wpdb, $gp_table_prefix;
+
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$wpdb->update(
+			"{$gp_table_prefix}event_attendees",
+			array( 'is_host' => $attendee->is_host() ? 1 : 0 ),
+			array(
+				'event_id' => $attendee->event_id(),
+				'user_id'  => $attendee->user_id(),
+			)
+		);
```

Comment:
We need to re-enable the rules we disabled above.


```suggestion
		);
		// phpcs:enable
```

Reply:
Done. Thank you!